### ISSUE TITLE
acl-public not working

### ISSUE THREAD

**Issue by Andreas-Halemba** [bloomreach/s4cmd#27]
i have set in my ./s3cfg acl_public = True, but my files are all not readable on my s3bucket i do a local to s3 sync. Any help?

**Comment by chouhanyang** [comment]
Not entirely sure what is acl_public = True is doing. Is it just set the ACL to public by default?

Currently, s4cmd doesn't have any capability to control ACL.

**Comment by Andreas-Halemba** [comment]
yes it does make the files readable for public. we are uploading a huge amount of images and your tool would be a huge improvement but it always makes the files not readable via browser. It is just an parameter you have to set in .s3cfg or at the cmd.

Are you planing to get it running with this sometime? 

THX

**Comment by liangrongze** [comment]
Me too.

**Comment by Andreas-Halemba** [comment]
Will we ever see this feature in s4cmd?

**Comment by chouhanyang** [comment]
With new pass-through API and boto3 library in s4cmd 2.0. Are you able to set the ACL now?

**Comment by bbbo** [comment]
thsu might work for uploads, mv/cp/sync etc, but s3cmd offers the option to set ACLs for existing objects by using s3cmd --setacl --acl-[private|public] s3://bucket/object , but only for single object or recursively for all objects.

Implementing a s4cmd touch command which does this for existing objects would be a HUGE advantage.
